README.md

```diff
@@ -1,19 +1,25 @@
 # React PropTypes Lab
 
 ## Objectives
-
 1. Practice using PropTypes declarations
 2. Describe when to use each declaration
 
-## Overview
+## A new job
+![Michael Scott](https://media.giphy.com/media/NXOF5rlaSXdAc/giphy.gif)
```

Comment:
Gif doesn't seem to be showing up. I reloaded several times. You seeing it @thomastuts?